### DISCUSSION TITLE
What's the isDefinitelyAnError?

### DISCUSSION THREAD

**Discussion by jupiter229** [remix-run/remix#6656] [Q&A]
I'm working in v2 migration, what I found was isDefinitelyAnError in the doc. 
https://remix.run/docs/en/main/pages/v2#catchboundary-and-errorboundary
![image](https://github.com/remix-run/remix/assets/68340397/3aa2daec-4945-4c72-9a6f-462f9f4aae3d)

It says "Cannot find name 'isDefinitelyAnError'" in my VS code, is this a custom function? How should I import it?

**Comment by AlemTuzlak** [comment]
I think from the comments above this function they are telling you to implement your own function that checks that the caught stuff IS an actual error

**Reply by kiliman** [reply]
Yes, something like:
```ts
function isDefinitelyAnError(error: unknown) {
  return error && (error instanceof Error || "message" in error)
}
```
